File: pinyougou-parent/pinyougou-shop-web/src/main/webapp/plugins/angularjs/service/typeTemplateService.js
app.service('typeTemplateService', function ($http) {

    this.findById = function (id) {
        return $http.get('../typeTemplate/findById.do?id=' + id);
    };

    this.findSpecListByTypeId = function (id) {
        return $http.get('../typeTemplate/findSpecListByTypeId.do?id=' + id);
    };
});

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/TypeTemplateService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbTypeTemplate;
import entity.PageResult;

import java.util.List;
import java.util.Map;

public interface TypeTemplateService {

    PageResult<TbTypeTemplate> findByPage(TbTypeTemplate typeTemplate, int pageNum, int pageSize) throws Exception;

    TbTypeTemplate findById(Long id) throws Exception;

    void save(TbTypeTemplate typeTemplate) throws Exception;

    void delete(Long[] ids) throws Exception;

    List<Map<String,Object>> getOptionList() throws Exception;

    List<Map> findSpecListByTypeId(Long id) throws Exception;
}